shimmingtoolbox/cli/mask.py

```diff
@@ -128,3 +129,65 @@ def threshold(fname_input, output, thr):
+                   " (os.curdir, 'mask.nii.gz')).")
+@click.option('-size', default='41',
+              help="(str): Size of the mask in the axial plane, given in pixel (Example: 35) or in millimeter "
+                   "(Example: 35mm). If shape=gaussian, size corresponds to sigma (Example: 45). (default: 41)")
+@click.option('-shape', type=click.Choice(['cylinder', 'box', 'gaussian']), default='cylinder',
+              help="(str): Shape of the mask. (default: cylinder)")
+@click.option('-contrast', type=click.Choice(['t1', 't2', 't2s', 'dwi']), default='t2s',
+              help="(str): Type of image contrast. Only with method=optic. (default: t1)")
+@click.option('-method', type=click.Choice(['optic', 'fitseg']), default='optic',
+              help="(str): Method used for extracting the centerline: "
+                   "- optic: automatic spinal cord detection method"
+                   "- fitseg: fit a regularized centerline on an already-existing cord segmentation. It will "
+                   "interpolate if slices are missing and extrapolate beyond the segmentation boundaries (i.e., every "
+                   "axial slice will exhibit a centerline pixel). (default: optic)")
+@click.option('-centerline_algo', type=click.Choice(['polyfit', 'bspline', 'linear', 'nurbs']), default='bspline',
+              help="(str): Algorithm for centerline fitting. Only relevant with -method fitseg (default: bspline)")
+@click.option('-centerline_smooth', default=30, help="(int): Degree of smoothing for centerline fitting. Only for "
+                                                     "-centerline-algo {bspline, linear}. (default: 30)")
+@click.option('-remove', type=click.IntRange(0, 1), default=1, help="(int): Remove temporary files. (default: 1)")
+@click.option('-verbose', type=click.IntRange(0, 2), default=1,
+              help="(int): Verbose: 0 = nothing, 1 = classic, 2 = expended. (default: 1)")
+def sct(fname_input, output, method, contrast, centerline_algo, centerline_smooth, size, shape, remove, verbose):
+
+    # Get the centerline
+    path_centerline = os.path.join(os.path.dirname(output), 'centerline')
+    if method == "optic":
+        run_subprocess(f"sct_get_centerline -i {fname_input} -c {contrast} -o {path_centerline} -v {str(verbose)}")
+
+    elif method == "fitseg" and (centerline_algo == "polyfit" or centerline_algo == "nurbs"):
+        run_subprocess(f"sct_get_centerline -i {fname_input} -method {method} -centerline-algo {centerline_algo} "
+                       f"-o {path_centerline} -v {str(verbose)}")
+
+    elif method == "fitseg" and (centerline_algo == "bspline" or centerline_algo == "linear"):
+        run_subprocess(f"sct_get_centerline -i {fname_input} -method {method} -centerline-algo {centerline_algo} "
+                       f"-centerline-smooth {str(centerline_smooth)} -o {path_centerline} -v {str(verbose)}")
+
+    else:
+        raise ValueError("Could not get centerline.")
```

Comment:
sure this is a value error? if it is, what value was wrong that caused it to fail? if it's not an input value error, pick one of these guys: https://docs.python.org/3/library/exceptions.html
If none of those fit we can custom make one